### ISSUE TITLE
PPASR入门级预测时softmax返回值为NAN

### ISSUE THREAD

**Issue by wangyhnudt** [yeyupiaoling/PaddlePaddle-DeepSpeech#110]
1）paddlepaddle2.1.3
2）飞腾2000 8G CPU
3）麒麟V10 aarch64 Ubuntu python3.6
模型信息：
1）PPASR语音识别
2）模型链接：https://github.com/yeyupiaoling/PPASR/tree/%E5%85%A5%E9%97%A8%E7%BA%A7
3）问题描述：在进行语音识别预测执行infer.py时，出现识别结果为空，查询时发现，out = paddle.nn.functional.softmax(out, 1)将tensor处理为NAN，tensor输出如下图：

![image](https://user-images.githubusercontent.com/43571945/141748726-44b094a1-6c2c-439b-86be-ad85be6251fc.png)

但是该问题在GPU rtx3090运行时，并没有出现。

**Comment by yeyupiaoling** [comment]
你来错地方了，这个是PaddlePaddle-DeepSpeech，PPASR在这里呢：https://github.com/yeyupiaoling/PPASR
不过我在Paddle上回复你了。

**Comment by wangyhnudt** [comment]
谢谢，我去看一下发自我的华为手机-------- 原始邮件 --------发件人： 夜雨飘零 ***@***.***>日期： 2021年11月15日周一 下午4:51收件人： yeyupiaoling/PaddlePaddle-DeepSpeech ***@***.***>抄送： wangyhnudt ***@***.***>, Author ***@***.***>主    题： Re: [yeyupiaoling/PaddlePaddle-DeepSpeech] PPASR入门级预测时softmax返回值为NAN (Issue #110)

你来错地方了，这个是PaddlePaddle-DeepSpeech，PPASR在这里呢：https://github.com/yeyupiaoling/PPASR

不过我在Paddle上回复你了。
